docs/src/user/config.rst

```diff
@@ -174,7 +174,7 @@ host
 ~~~~
 
 :Type: str
-:Default: 127.0.1.1
+:Default: orion_db.pkl
```

Comment:
The default is https://github.com/Epistimio/orion/blob/develop/src/orion/core/io/database/pickleddb.py#L25

Reply:
Maybe but trying to import that file in the config function leads to an import loop and fails.